.gitattributes

```diff
@@ -0,0 +1 @@
+*.csv filter=lfs diff=lfs merge=lfs -text
```

Comment:
Um, I am sure this is correct, but, what does it do?

Reply:
It specifies which files to automatically store/retrieve as LFS, and a few other settings. This is the default recommended, and I specified storing all *.csv files.